cases.ts

```diff
@@ -111,7 +111,7 @@ export const rule4Tree: ASTNode = {
           name: 'EQ',
           args: [
             { type: 'path', path: './companyType' },
-            { type: 'value', value: 'nffe' },
+            { type: 'value', value: "'nffe'" },
```

Comment:
Probably it's better to introduce the `dataType` property here with possible values 'boolean', 'string', 'number', 'array'. What do you think?

Reply:
I changed this to preserve the quotes in strings and arrays entered by the user, for example if is entered 'nffe', "nffe" or "nffe (incomplete, without last quote)